### ISSUE TITLE
`nthChild`

### ISSUE THREAD

**Issue by rjhilgefort** [SentiaAnalytics/bs-css#116]
How possible is it to support `nthChild` selectors? If you point me to where to do so, I would try to contribute.

**Comment by wegry** [comment]
It would probably function similarly to https://github.com/SentiaAnalytics/bs-css/blob/master/src/Css.rei#L975

**Comment by remithomas** [comment]
You could use `selector` to do the same. But I agree it is not completely safe. I'm waiting the merge of your PR #121 

```re
let card = style([
   selector(
      ":nth-child(1)",
      [
        left(em(0.))
      ],
    ),
]);
```

**Comment by giraud** [comment]
implemented in 10.0.1
